Website/js/Debug/dnn.modalpopup.js

```diff
@@ -191,9 +191,7 @@
                     closingUrl: closingUrl,
 					open: dialogOpened,
                     close: function() { window.dnnModal.closePopUp(refresh, closingUrl); }
-                })
```

Comment:
these lines where making impacts on loading of popup modal on first load and on subsequent minimized calls no impact so different UX experience for a users.